src/api/searchApi.ts

```diff
@@ -93,3 +93,15 @@ export const fetchNviCandidate = async (identifier: string) => {
 
   return getNviCandidates.data;
 };
+
+export const fetchResults2 = async (results: number, from: number, { doi = '' }) => {
+  const paginationQuery = `results=${results}&from=${from}`;
+  const doiQuery = doi ? `doi=${doi}` : '';
```

Comment:
gir det ikke mer mening at vi legger til fnuttene her? i stedet for at man må sende inn en "strenget streng" alle plasser hvor vi vil bruke dette
```suggestion
  const doiQuery = doi ? `doi="${doi}"` : '';
```

Reply:
Jo, absolutt